lib/git_reflow.rb

```diff
@@ -99,8 +99,10 @@ def deliver(options = {})
 
           if committed
             say "Merge complete!", :success
-            deploy_and_cleanup = ask "Would you like to push this branch to your remote repo and cleanup your feature branch? "
-            if deploy_and_cleanup =~ /^y/i
+            always_deploy_and_cleanup = GitReflow::Config.get('reflow.always-deploy-and-cleanup').strip == "true"
+            deploy_and_cleanup = always_deploy_and_cleanup || HighLine.agree("Would you like to push this branch to your remote repo and cleanup your feature branch?")
```

Comment:
Spec issue: We're only stubbing the `ask` method on HighLine for our tests.  If you want to keep `agree` here, you'll have to stub that method as well.  It may be easier to just use `ask` here.  Also, you don't need to prefix the method with `HighLine`.  You can just call `ask` or `agree` directly in this context.